packages/core/src/components/button/buttons.tsx

```diff
@@ -13,20 +13,30 @@ import * as React from "react";
 
 import * as Classes from "../../common/classes";
 import { IActionProps, removeNonHTMLProps } from "../../common/props";
+import { Spinner } from "../spinner/spinner";
 
 export interface IButtonProps extends IActionProps {
     /** A ref handler that receives the native HTML element backing this component. */
     elementRef?: (ref: HTMLElement) => any;
 
     /** Name of icon (the part after `pt-icon-`) to add to button. */
     rightIconName?: string;
+
+    /**
+     * If set to true, the button will display a centered loading spinner instead of the button contents.
+     * The size of the button is not affected by the value of this prop.
```

Comment:
I can change, although work noting that it is significant that the height doesn't change too :)